src/index.js

```diff
@@ -18,6 +18,7 @@ function CredentialCommons() {
   this.isClaimRelated = isClaimRelated;
   this.services = services;
   this.errors = errors;
+  this.headers = headers;
```

Comment:
Maybe “constants” here - so we can get “constants.headers” instead of “headers.headers”.

Reply:
good idea

Reply:
updated